lldb/include/lldb/API/SBStatisticsOptions.h

```diff
@@ -24,6 +24,9 @@ class LLDB_API SBStatisticsOptions {
 
   void SetSummaryOnly(bool b);
   bool GetSummaryOnly();
+  
+  void SetForceLoading(bool b);
+  bool GetForceLoading();
```

Comment:
The `SetForceLoading` name doesn't clearly indicate to us what we are force loading just by the API name. How about:
```
/// If set to true, the debugger will load all debug info that is available
/// and report statistics on the total amount. If this is set to false, then
/// only report statistics on the currently loaded debug information. 
/// This can avoid loading debug info from separate files just so it can
/// report the total size which can slow down statistics reporting.
void SetReportAllAvailableDebugInfo(bool b);
bool SetReportAllAvailableDebugInfo();
```
This setting would default to `false`.